.github/workflows/chaos-mesh.yml

```diff
@@ -0,0 +1,222 @@
+        kubectl logs dm-master-0 || true
+        echo "<<<<< show previous log for dm-master-0 >>>>>"
+        kubectl logs dm-master-0 -p || true
+        echo "<<<<< show current log for dm-master-1 >>>>>"
+        kubectl logs dm-master-1 || true
+        echo "<<<<< show previous log for dm-master-1 >>>>>"
+        kubectl logs dm-master-1 -p || true
+        echo "<<<<< show current log for dm-master-2 >>>>>"
+        kubectl logs dm-master-2 || true
+        echo "<<<<< show previous log for dm-master-2 >>>>>"
+        kubectl logs dm-master-2 -p || true
+
+    - name: Set up DM-worker
+      run: |
+        kubectl apply -f $GITHUB_WORKSPACE/chaos/manifests/dm-worker.yaml
+        kubectl get -f $GITHUB_WORKSPACE/chaos/manifests/dm-worker.yaml
+        kubectl describe -f $GITHUB_WORKSPACE/chaos/manifests/dm-worker.yaml
+    # NOTE: even some DM-worker instances are not ready, we still continue and let chaos test cases to check again.
+    - name: Wait for DM-worker ready
+      run: |
+        sleep 10
+        kubectl wait --for=condition=Ready pod -l app=dm-worker --all --timeout=120s || true
+        echo "<<<<< show pvc >>>>>"
+        kubectl get pvc -l app=dm-worker -o wide
+        echo "<<<<< show pv >>>>>"
+        kubectl get pv -o wide
+        echo "<<<<< show svc >>>>>"
+        kubectl get svc -l app=dm-worker -o wide
+        echo "<<<<< show sts >>>>>"
+        kubectl get sts -l app=dm-worker -o wide
+        echo "<<<<< show po >>>>>"
+        kubectl get po -l app=dm-worker -o wide
+        echo "<<<<< describe po >>>>>"
+        kubectl describe po -l app=dm-worker
+        echo "<<<<< describe pvc >>>>>"
+        kubectl describe pvc -l app=dm-worker
+        echo "<<<<< show current log for dm-worker-0 >>>>>"
+        kubectl logs dm-worker-0 || true
+        echo "<<<<< show previous log for dm-worker-0 >>>>>"
+        kubectl logs dm-worker-0 -p || true
+        echo "<<<<< show current log for dm-worker-1 >>>>>"
+        kubectl logs dm-worker-1 || true
+        echo "<<<<< show previous log for worker-master-1 >>>>>"
+        kubectl logs dm-worker-1 -p || true
+        echo "<<<<< show current log for dm-worker-2 >>>>>"
+        kubectl logs dm-worker-2 || true
+        echo "<<<<< show previous log for dm-worker-2 >>>>>"
+        kubectl logs dm-worker-2 -p || true
+
+    # NOTE: we sleep a while when check members ready in cases before applying any chaos operations.
+    - name: Set up chaos test cases
+      run: |
+        kubectl apply -f $GITHUB_WORKSPACE/chaos/manifests/cases.yaml
+        kubectl get -f $GITHUB_WORKSPACE/chaos/manifests/cases.yaml
+        kubectl describe -f $GITHUB_WORKSPACE/chaos/manifests/cases.yaml
+        sleep 60
+
+    - name: Encode chaos-mesh action
+      run: |
+        echo "::set-env name=CFG_BASE64::$(base64 -w 0 $GITHUB_WORKSPACE/chaos/manifests/pod-failure-dm.yaml)"
+
+    - name: Run chaos mesh action
+      uses: chaos-mesh/chaos-mesh-action@master
```

Comment:
Not sure where GitHub action calls `chaos/cases`

Reply:
`uses: chaos-mesh/chaos-mesh-action@master` only applis chaos operations (defined in `./chaos/manifests/pod-failure-dm.yaml`) into the cluster.

`chaos/cases` is called by `./chaos/manifests/cases.yaml` (see https://github.com/pingcap/dm/blob/87f60a3cf33f1fe588d9b420e8169124934d66f9/chaos/manifests/cases.yaml#L25-L27) ( and `cases.yaml`  is applied in `kubectl apply -f $GITHUB_WORKSPACE/chaos/manifests/cases.yaml`).